From 414a2def11e52749ba6e5e45c6cf9aba012bc079 Mon Sep 17 00:00:00 2001
From: nobody <nobody@nowhere.com>
Date: Sun, 14 May 2023 20:57:10 +0800
Subject: [PATCH] replace TEE in makefile

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ad83d60dc3..b4f52fd58c 100644
--- a/Makefile
+++ b/Makefile
@@ -1327,7 +1327,7 @@ cmd_binman = $(srctree)/tools/binman/binman $(if $(BINMAN_DEBUG),-D) \
 		-I arch/$(ARCH)/dts -a of-list=$(CONFIG_OF_LIST) \
 		$(foreach f,$(BINMAN_INDIRS),-I $(f)) \
 		-a atf-bl31-path=${BL31} \
-		-a tee-os-path=${TEE} \
+		-a tee-os-path=$(srctree)/tee.bin \
 		-a opensbi-path=${OPENSBI} \
 		-a default-dt=$(default_dt) \
 		-a scp-path=$(SCP) \
-- 
2.39.2 (Apple Git-143)
